### ISSUE TITLE
Somehow i get instantly disconnected when trying to connect my server

### ISSUE THREAD

**Issue by TheButterbrotMan** [jexline/exline-mods-issues#24]
Singleplayer works. Could identify two mods (Doors & Furniture) as cause by using binary. There is nothing in the .logs, only TheButterbrotMan disconnected.

Problem is present in both latest versions. Experienced a similar problem yesterday:

**Comment by TheButterbrotMan** [comment]
https://github.com/ianm1647/expandeddelight/issues/6

**Comment by TheButterbrotMan** [comment]
I also having a heavly modded enviroment. So probably something interrupts with some mod.

**Comment by jexline** [comment]
Sorry I have been gone, But have you still had issues? I have tested on my own server but if you are using a heavily modded environment something might be conflicting. I will try to help figure it out if you still need help.

**Comment by TheButterbrotMan** [comment]
Yes, im still experiencing this issue. Im not at home at the moment. Cant really work on this issue sadly.
